src/components/apiTokens/CreateAPIToken.tsx

```diff
@@ -207,21 +212,45 @@ function CreateAPIToken({
         setAdminPermission(e.target.value)
     }
 
+    const handleQuestion = () => {
```

Comment:
If its possible create common function somewhere else & use on both the place

Reply:
done